packages/components/inputs/rich-text-utils/src/rich-text-body/rich-text-body.tsx

```diff
@@ -36,10 +41,115 @@ import { MARK_TAGS, BLOCK_TAGS } from '../tags';
 import hasBlock from '../has-block';
 import messages from './messages';
 
+type TMoreStylesDropdownItem = {
+  value?: string;
+  children?: ReactNode;
+};
+
+type TDropdownLabel = {
+  children?: ReactNode;
+};
+
+type TStylesDropdownItem = {
+  value?: string;
+  children?: ReactNode;
+  displayName?: string;
+};
+
+type TStyleDropdownOptionParagraph = {
+  id: string;
+  description: string;
+  defaultMessage: string;
+};
+
+type TStyleDropdownOptions = {
+  formatMessage: (message: TStyleDropdownOptionParagraph) => string;
+};
+
+type TNodeRefObject = {
+  clientHeight: number;
+} & LegacyRef<HTMLDivElement>;
+
+type TRef = {
+  registerContentNode: TNodeRefObject;
+  containerRef?: LegacyRef<HTMLDivElement>;
+};
+
+type TType = {
+  type: string;
+};
+
+type TEditor = {
```

Comment:
I noticed we are using the editor in a bunch of this `rich-text-utils` files and we're typing it everywhere is needed (only with the slice of attributes used in each context).

I was wondering if it would be more maintainable if we have a shared types in this package where we precisely type the editor and use that type everywhere across this package files.

Reply:
That's a good idea. Will do that.